src/content/en/updates/2017/03/devtools-release-notes.md

```diff
@@ -0,0 +1,166 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: New features and changes coming to DevTools in Chrome 58.
+
+{# wf_updated_on: 2017-03-03 #}
+{# wf_published_on: 2017-03-03 #}
+{# wf_tags: chrome58,devtools #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: New features and changes coming to DevTools in Chrome 58. #}
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+# What's New In DevTools (Chrome 58) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+Welcome to the first installment of the DevTools release notes! From here on
+out, the first time you open a new version of Chrome, DevTools opens the
+**What's New** drawer with a link to the release notes for that version.
```

Comment:
This is massively cool. How are the links being managed? I'm worried about them breaking.

@petele